mysk_utils/schema/student_teacher/person.py

```diff
@@ -0,0 +1,62 @@
+from pydantic import BaseModel
+from typing import List, Optional
+from datetime import datetime, date
+from enum import Enum, IntEnum
+
+from .contacts import Contact, QueryContact
+
+
+class ThaiPrefix(Enum):
+    """
+    Thai prefix for name
+    """
+
+    Master = "เด็กชาย"
+    Mr = "นาย"
+    Mrs = "นาง"
+    Miss = "นางสาว"
+
+
+class EnglishPrefix(Enum):
+    """
+    English prefix for name
+    """
+
+    Mr = "Mr."
+    Mrs = "Mrs."
+    Miss = "Miss."
+    Master = "Master."
```

Comment:
I don't think there's a period after master here.
(It looks like “Master.” appears multiple times, so if you wanna change this just be sure to change them all!)